doc/attributes.md

```diff
@@ -0,0 +1,51 @@
+# User And Group Attributes
+
+This document describes the attributes that can be indicated for users and groups, either through the YaST UI of the *users* client or through AutoYaST. Each attribute should have a counterpart in the new *Y2Users* code. This correspondence helps to translate the users data structure from *Yast::Users* perl module to the new code.
+
+## User
+
+The YaST UI for creating and editing a user offers the following attributes:
+
+| UI Field | Y2Users attr | Only when adding | Only when editing | Description |
+| :--- | :--- | :--- | :--- | :--- |
+| full name | gecos | | | |
+| name | name | | | |
+| password | password | | | |
+| system mail | | | | |
+| disable login | * password starting by !$ | | | |
+| uid | uid | | | |
+| home dir | home | | | | |
+| home dir permission | | | | |
+| empty home | | yes | | |
+| Move to new location | | | yes |  |
+| Btrfs subvolume | btrfs_subvolume_home | | | |
+| Additional info | gecos | | | |
+| login shell | shell | | | |
+| group | primary_group | | | |
+| additional groups | groups | | | |
+
+### Home management
+
+### Creating a user
+
+* A user can be created with or without home.
+* To create a user without home, the *home dir* path should be set to empty.
```

Comment:
That's true... and tricky.

Out of the scope of this document, of course, but I'd fix the UI to make it explicit. Right now, if user clean the _home dir_ but goes back to the _User Data_ tab and forward to _Details_ tab... the *home dir* is auto-filling again.